Pasteboard Viewer/ContentsScreen.swift

```diff
@@ -258,3 +320,67 @@ extension Pasteboard.Type_ {
 		return (utType, data)
 	}
 }
+
+struct HexView: NSViewRepresentable {
+	class Coordinator: HFRepresenter, ObservableObject {
```

Comment:
```suggestion
	class Coordinator: HFRepresenter {
```

It doesn't look like it's used for anything.